packages/sdk-react/src/components/approvalRequests/ApprovalRequestsTable/UserCell/UserCell.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import { UserAvatar } from '@/components/UserAvatar/UserAvatar';
+import { useEntityUserById } from '@/core/queries';
+import { Chip, Skeleton } from '@mui/material';
+
+interface UserCellProps {
+  entityUserId: string;
+}
+
+export const UserCell = ({ entityUserId }: UserCellProps) => {
+  const { data: entityUser, isLoading } = useEntityUserById(entityUserId);
+
+  if (!entityUser) {
+    return null;
+  }
+
+  if (isLoading) {
+    return (
+      <Chip
```

Comment:
Maybe, makes sense to move it to Suspense?

Reply:
_***Lost in thread_
@vkuprin, by the way, normally, in components we prefer not to use Suspense because it requires an extra wrapper (`<Suspense/>`).

Reply:
I agree with @radist2s.
@vkuprin you can add your idea to our frontend sync agenda and we will talk about how and where to use suspense and the benefits for our project.
I will close it for now.

Reply:
Gotcha, let's discuss this on fe sync then